src/app/plugins.js

```diff
@@ -0,0 +1,130 @@
+import { fromEntries } from '../utils';
+import { CoordinationType } from './constants';
+import {
+  COMPONENT_COORDINATION_TYPES,
+  DEFAULT_COORDINATION_VALUES,
+} from './state/coordination';
+
+export const PLUGIN_VIEW_TYPES_KEY = '__VITESSCE_PLUGIN_VIEW_TYPES__';
+export const PLUGIN_COORDINATION_TYPES_KEY = '__VITESSCE_PLUGIN_COORDINATION_TYPES__';
+export const PLUGIN_COORDINATION_TYPES_PER_VIEW_KEY = '__VITESSCE_PLUGIN_COORDINATION_TYPES_PER_VIEW__';
+export const PLUGIN_FILE_TYPES_KEY = '__VITESSCE_PLUGIN_FILE_TYPES__';
+
+// Reference: https://github.com/higlass/higlass-register/blob/master/src/index.js
+// TODO: rather than the registration functions and storing things on the window object,
+// should plugins be provided as props to the main <Vitessce/> React component
+// and stored via some React context?
```

Comment:
If it works for higlass, it works for me tbh.  And if it breaks, we all have to deal with it.